extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -483,48 +485,54 @@ public function setData( ) {
 					wfRunHooks( 'CuratedContentSave', [ $sections ] );
 				}
 			}
+			$this->response->setVal( 'status', $status );
 
+		} else {
+			$this->response->setCode( \Wikia\Service\ForbiddenException::CODE );
+			$this->response->setVal( 'message', 'No permissions to save curated content' );
```

Comment:
Question: Why you're returning message if you aren't reading it on front-end side?